org.ekstep.questionunit.mcq-1.1/renderer/template/mcq-layouts.js

```diff
@@ -462,7 +462,7 @@ MCQController.grid2.getOption = function (option, key) {
       src=<%=MCQController.pluginInstance.getAssetUrl(option.image) %> />\
   <%}%>\
   <%if(!option.image && option.text){%>\
-    <div class='mcq2-2-option-text'><%= option.text %></div>\
+    <div class='mcq2-2-option-text option-text-common-questionunit-plugins'><%= option.text %></div>\
```

Comment:
Make it class name small option-text-common-questionunit-plugins => common-option-text

Reply:
Fixed as part of 
https://github.com/project-sunbird/sunbird-content-plugins/pull/139/